packages/admin-ui/src/filters/filter/filter.state.tsx

```diff
@@ -11,9 +12,10 @@ import { useFilterStatus } from '../use-filter-status'
 export function useFilterState<T extends AnyObject>(
   props?: UseFilterStateProps<T>
 ): UseFilterStateReturn<T> {
-  const { fullList = [] } = props || {}
+  const { fullList: initialFullList = [] } = props || {}
```

Comment:
fullList represents the initial matches? What's the difference between matches, setMatches and fullList, setFullList?

Reply:
fullList => list that is searched by combobox internally
matches => current matches from the fulllist. If the user is controlling everything they can ignore fulllist and just set matches in general

setFullList => you can use it to set the searchable list, and then let the component perform the searches
setMatches => you are the one controlling the searches and so you only set the "search result" everytime